clone-project.sh

```diff
@@ -10,15 +10,15 @@ if [[ -d $slug ]]; then
     cd $slug
     git checkout $sha
     echo "SHA is $(git rev-parse HEAD)"
-elif [[ ! -f "$input_container/$modified_slug_sha_module.zip" ]]; then
+elif [[ ! -f "$input_container/projects/$modified_slug_sha_module.zip" ]]; then
     echo "$modified_slug_sha_module doesn't exist in the container"
     git clone https://github.com/$slug $slug
     cd $slug
     git checkout $sha
     echo "SHA is $(git rev-parse HEAD)"
 else
     echo "$modified_slug_sha_module already exists in the container"
-    cp $input_container/$modified_slug_sha_module.zip .
+    cp $input_container/projects/$modified_slug_sha_module.zip .
```

Comment:
Dependencies should be saved in install-project.sh if the $modified_slug_sha_module does not exist. If it exist, then the dependencies should be copied and unzipped into a directory on the local machine and not the input container.